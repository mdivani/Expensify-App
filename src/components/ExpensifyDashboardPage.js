import React from 'react';
import ExpensesList from './ExpensesList';
import ExpenseListFilter from './ExpenseListFilter';
import ExpensesSummary from './ExpensesSummary';

const ExpensifyDashboardPage = () => (
    <div>
        <ExpenseListFilter />
        <ExpensesList />
        <ExpensesSummary />
    </div>
);

export default ExpensifyDashboardPage;