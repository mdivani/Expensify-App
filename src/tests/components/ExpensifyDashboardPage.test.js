import React from 'react';
import { shallow } from 'enzyme';
import ExpensifyDashboardPage from '../../components/ExpensifyDashboardPage';

test('should render page not found error correctly', () => {
    const wrapper = shallow(<ExpensifyDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});