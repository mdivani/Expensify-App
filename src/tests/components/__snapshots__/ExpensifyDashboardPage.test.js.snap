// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render page not found error correctly 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
  <Connect(ExpensesSummary) />
</div>
`;
